testplan/testing/multitest/result.py

```diff
@@ -1212,40 +1207,31 @@ def match(
             )
 
         :param actual: Original FIX message.
-        :type actual: ``dict``
         :param expected: Expected FIX message, can include compiled
                          regex patterns or callables for
                          advanced comparison.
-        :type expected: ``dict``
+        :param include_only_tags_of_expected: Use the tags present in the expected message.
         :param include_tags: Tags to exclusively consider in the comparison.
-                             It will automatically use the tags from the expected message when set to True.
-        :type include_tags: ``list`` of ``object`` (items must be hashable) or ``bool``
         :param exclude_tags: Keys to ignore in the comparison.
-        :type exclude_tags: ``list`` of ``object`` (items must be hashable)
         :param report_mode: Specify which comparisons should be kept and
                             reported. Default option is to report all
                             comparisons but this can be restricted if desired.
                             See ReportOptions enum for more detail.
-        :type report_mode: ``testplan.common.utils.comparison.ReportOptions``
         :param actual_description: Column header description for original msg.
-        :type actual_description: ``str``
         :param expected_description: Column header
                                      description for expected msg.
-        :type expected_description: ``str``
         :param description: Text description for the assertion.
-        :type description: ``str``
         :param category: Custom category that will be used for summarization.
-        :type category: ``str``
 
         :return: Assertion pass status
-        :rtype: ``bool``
         """
         entry = assertions.FixMatch(
             value=actual,
             expected=expected,
             description=description,
             category=category,
-            include_tags=expected if include_tags is True else include_tags,
+            include_only_tags_of_expected=include_only_tags_of_expected,
```

Comment:
`expected_keys_only`, `rhs_keys_only`. These are my only natural simplifications for the name at this point. Perhaps something starting with `is` due to the boolean, so say `is_rhs_keys_only`.